pkg/ostreeconfig/ostreeconfig_test.go

```diff
@@ -0,0 +1,81 @@
+package ostreeconfig
+
+import (
+	"os"
+	"strings"
+	"testing"
+)
+
+func TestBasePathNull(t *testing.T) {
+	otc := OstreeConfig{}
+
+	otc.BasePath = ""
+	otc.FullPath = "/tmp/repo"
+
+	err := otc.InitRepo()
+	if strings.Compare(err.Error(), "BasePath must not be empty") != 0 {
+		t.Errorf("%s", err)
+	} else if err == nil {
+		t.Errorf("No error, should have failed")
+		return
+	}
+}
+
+func TestFullPathNull(t *testing.T) {
+	otc := OstreeConfig{}
+
+	otc.BasePath = "/tmp"
+	otc.FullPath = ""
+
+	err := otc.InitRepo()
+	if strings.Compare(err.Error(), "FullPath must not be empty") != 0 {
+		t.Errorf("%s", err)
```

Comment:
Is it necessary to use the formatter here? Couldn't it just be `t.Error(err)`, not to nit but it seems like the formatter is just added bulk